server/src/main/java/dev/slimevr/protocol/rpc/settings/RPCSettingsHandler.java

```diff
@@ -123,11 +129,31 @@ public void onChangeSettingsRequest(GenericConnection conn, RpcMessageHeader mes
 			}
 		}
 
+		if (req.tapDetectionSettings() != null) {
+			TapDetectionConfig tapDetectionConfig = this.api.server
+				.getConfigManager()
+				.getVrConfig()
+				.getTapDetection();
+			var tapDetectionSettings = req.tapDetectionSettings();
+
+			if (tapDetectionSettings != null) {
+				tapDetectionConfig.setEnabled(tapDetectionSettings.tapResetEnabled());
```

Comment:
This should check `tapDetectionConfig.hasTapResetEnabled()` if it's available, otherwise just resolve this comment.

Reply:
Am I assuming you meant to say ```tapDetectionSettings.hasTapResetEnabled()```? If you look at the other boolean values in toggles they also don't have this check because if the value is false  ```tapDetectionConfig.hasTapResetEnabled()``` will also return false.

Reply:
Ah I see alright, just a little quirky